masterlist.yaml

```
@@ -17245,18 +16991,13 @@ plugins:
     after: [ 'ESFCompanions.esp' ]
     msg:
       - <<: *alreadyInX
-        condition: 'version("Requiem.esp", "2.0.0", >=)'
+        condition: 'active("Requiem.esp")'
         subs: [ 'Requiem.esp' ]
```

To keep the masterlists' style consistent, we also prefer to order the `subs` key before the `condition` key.

```suggestion
        subs: [ 'Requiem.esp' ]
        condition: 'active("Requiem.esp")'
```